prime-angular-frontend/src/app/modules/site-registration/pages/business-licence/business-licence.component.ts

```diff
@@ -110,6 +138,30 @@ export class BusinessLicenceComponent implements OnInit {
     }
   }
 
+  public downloadBusinessLicence(event: Event) {
+    event.preventDefault();
+    this.siteResource.getBusinessLicenceDocumentToken(this.siteService.site.id)
+      .subscribe((token: string) =>
+        this.utilsService.downloadToken(token)
+      );
+  }
+
+  public toggleDeferred() {
+    if (this.deferredLicence) {
```

Comment:
Looks like you're removing the validators, but not clearing the values.  Do you want to keep the values and have them submitted if the toggle is unchecked?  If you don't I would look at the `FormUtilsService` and use the `setValidators` and `resetAndClearValidators` methods.